examples/fem_system/fem_system_ex1/naviersystem.C

```diff
@@ -549,29 +555,30 @@ bool NavierSystem::mass_residual (bool request_jacobian,
 
   for (unsigned int qp = 0; qp != n_qpoints; ++qp)
     {
-      Number u = c.interior_value(u_var, qp),
-        v = c.interior_value(v_var, qp),
-        w = c.interior_value(w_var, qp);
+      Number u_dot, v_dot, w_dot;
+      c.interior_rate(u_var, qp, u_dot),
+      c.interior_rate(v_var, qp, v_dot),
+      c.interior_rate(w_var, qp, w_dot);
 
       // We pull as many calculations as possible outside of loops
       Number JxWxRe   = JxW[qp] * Reynolds;
-      Number JxWxRexU = JxWxRe * u;
-      Number JxWxRexV = JxWxRe * v;
-      Number JxWxRexW = JxWxRe * w;
+      Number JxWxRexU = JxWxRe * u_dot;
+      Number JxWxRexV = JxWxRe * v_dot;
+      Number JxWxRexW = JxWxRe * w_dot;
 
       for (unsigned int i = 0; i != n_u_dofs; ++i)
         {
-          Fu(i) += JxWxRexU * phi[i][qp];
-          Fv(i) += JxWxRexV * phi[i][qp];
+          Fu(i) -= JxWxRexU * phi[i][qp];
+          Fv(i) -= JxWxRexV * phi[i][qp];
           if (dim == 3)
-            Fw(i) += JxWxRexW * phi[i][qp];
+            Fw(i) -= JxWxRexW * phi[i][qp];
```

Comment:
Do you have some screenshots of this example's solutions from before/after the sign change? It seems like the results should have been very wrong before...